types/knockout/index.d.ts

```diff
@@ -660,9 +660,9 @@ declare namespace KnockoutComponentTypes {
 
     interface Loader {
         getConfig? (componentName: string, callback: (result: ComponentConfig | null) => void): void;
-        loadComponent? (componentName: string, config: ComponentConfig, callback: (result: Definition) => void): void;
-        loadTemplate? (componentName: string, templateConfig: any, callback: (result: Node[]) => void): void;
-        loadViewModel? (componentName: string, viewModelConfig: any, callback: (result: any) => void): void;
+        loadComponent? (componentName: string, config: ComponentConfig, callback: (result: Definition | null) => void): void;
+        loadTemplate? (componentName: string, templateConfig: any, callback: (result: Node[] | null) => void): void;
+        loadViewModel? (componentName: string, viewModelConfig: any, callback: (result: any | null) => void): void;
```

Comment:
I added it such they look like the same and because `{} | null` would make more sense for me instead of `any`, but I never used Knockout and therefore am not sure.